### PR TITLE
services: added bulk 'add to parent' call

### PR DESCRIPTION
**Pull request by alejandromumo**
closes https://github.com/inveniosoftware/invenio-rdm-records/issues/1677

### CODE REVIEW

File: invenio_communities/communities/services/service.py [hunk 1]
```diff
@@ -714,6 +714,25 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
...
+        :param comm_slugs: The list of community slugs to add to the new parent.
+        :param new_parent: The slug of the new parent community.
+        """
+        t_comm = self.read(identity, new_parent)
+        for c_slug in comm_slugs:
+            c_comm = self.read(identity, c_slug)
+            self.update(
+                identity,
+                c_slug,
+                data={**c_comm.data, "parent": {"id": t_comm.id}},
```
Comment (jrcastro2) [inline invenio_communities/communities/services/service.py:731]: Question: Should there be a check for an already existing parent, or is it okay to replace it? (maybe adding a force option to replace would be useful (?))

To us it looks like if we want to allow updates at least we should call it `bulk_update_parent` instead.

File: invenio_communities/communities/services/service.py [hunk 2]
```diff
@@ -714,6 +714,25 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
 
         return True
 
+    @unit_of_work()
+    def bulk_add_to_parent(self, identity, comm_slugs, new_parent, uow=None):
```
Comment (jrcastro2) [inline invenio_communities/communities/services/service.py:718]:
```suggestion
    def bulk_add_parent(self, identity, comm_slugs, new_parent, uow=None):
```

**Review by jrcastro2 — COMMENTED** [review]
Pair-reviewed with @0einstein0

File: invenio_communities/communities/services/service.py (hunk 2 above)
Comment (alejandromumo) [reply invenio_communities/communities/services/service.py:718]: I disagree on this one, semantically `community_service.bulk_add_parent` seems it's adding parents in bulk. We're adding records, in bulk, to one parent (`community_service.bulk_add_to_parent)`. But I'm open to other suggestions.

File: invenio_communities/communities/services/service.py (hunk 1 above)
Comment (alejandromumo) [reply invenio_communities/communities/services/service.py:731]: Regarding the questions:

1) will be added, I don't see why not. 
2) I also agree with this one. The way I saw it, we needed a way to "transfer" multiple communities to a new parent. Thus, the wording `add to parent`. But you're right, this is updating the parent (not adding).

File: invenio_communities/communities/services/service.py [hunk 3]
```diff
@@ -714,6 +714,25 @@ def on_relation_update(self, identity, record_type, records_info, notif_time):
 
         return True
 
+    @unit_of_work()
+    def bulk_update_parent(self, identity, comm_slugs, new_parent, uow=None):
```
Comment (slint) [inline invenio_communities/communities/services/service.py:718]:
```suggestion
    def bulk_update_parent(self, identity, community_ids, parent, uow=None):
```

minor: slugs and UUIDs are resolved uniformly in communities, so it's ok to be "generic" about it :)

Also some renaming to simplify.

**Review by slint — COMMENTED** [review]
LGTM, minor namings nits (I will apply and merge)

**Pull request merged**